docs/tutorials/integrations/kserve/kserve-grpc.md

```diff
@@ -0,0 +1,94 @@
+---
+template: main.html
+---
+
+# Load Test a KServe Model (via gRPC)
+
+This tutorial shows how easy it is to run a load test for KServe when using gRPC to make requests. We use a scikit-learn model to demonstrate. The same approach works for any model type. 
+
+???+ "Before you begin"
+    1. Try [your first experiment](../../../getting-started/your-first-experiment.md). Understand the main [concepts](../../../getting-started/concepts.md) behind Iter8 experiments.
+    2. Ensure that you have the [kubectl](https://kubernetes.io/docs/reference/kubectl/) CLI.
+    3. Have access to a cluster running [KServe](https://kserve.github.io/website). You can create a [KServe Quickstart](https://kserve.github.io/website/0.10/get_started/#before-you-begin) environment as follows:
+    ```shell
+    curl -s "https://raw.githubusercontent.com/kserve/kserve/release-0.10/hack/quick_install.sh" | bash
+
+## Deploy an InferenceService
+
+Create an InferenceService which exposes a gRPC port. The following serves the SciKit [irisv2 model](https://kserve.github.io/website/0.10/modelserving/v1beta1/sklearn/v2/#deploy-with-inferenceservice):
+
+```shell
+cat <<EOF | kubectl create -f -
+apiVersion: "serving.kserve.io/v1beta1"
+kind: "InferenceService"
+metadata:
+  name: "sklearn-irisv2"
+spec:
+  predictor:
+    model:
+      modelFormat:
+        name: sklearn
+      runtime: kserve-mlserver
+      protocolVersion: v2
+      storageUri: "gs://seldon-models/sklearn/mms/lr_model"
+      ports:
+      - containerPort: 9000
+        name: h2c
+        protocol: TCP
+EOF
+```
+
+***
+
+## Launch Experiment
+
+Launch the Iter8 experiment inside the Kubernetes cluster:
+
+```shell
+ISVC=sklearn-irisv2
+INGRESS_HOST=$(kubectl get isvc $ISVC -o jsonpath='{.status.components.predictor.address.url}' | sed 's#.*//##')
```

Comment:
Are these really INGRESS hosts and ports? Or are they gRPC hosts and ports?

Hopefully, the predictor URL is cluster local, and hence we can rename them as gRPC hosts and ports instead of ingress?